packages/block-library/src/navigation/edit/index.js

```diff
@@ -690,6 +696,51 @@ function Navigation( {
 		/>
 	);
 
+	const NAVIGATION_SIDEBAR_NAME = 'edit-site/block-inspector';
```

Comment:
Are we happy for this to be intended _not_ to work in other editors?

Reply:
No, I think it should work in both. I'm not sure how to achieve that.

Reply:
Presumably the Post Editor also has a sidebar right? We should toggle that one if it's available.

Reply:
It does with my latest changes :)